tests/containers/docker_3rd_party_images.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Package: docker
+# Summary: Pull and test several base images (alpine, openSUSE, debian, ubuntu, fedora, centos, ubi) for their base functionality
+#          Log the test results in docker-3rd_party_images_log.txt
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils;
+use containers::common;
+use containers::urls 'get_3rd_party_images';
+use containers::container_images qw(test_3rd_party_image upload_3rd_party_images_logs);
+use registration;
+
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    my ($running_version, $sp, $host_distri) = get_os_release;
+    my $runtime = 'docker';
```

Comment:
why do we need to keep this variable at all? You can replace now all the instances of this variable by its value.

Reply:
I saw similar pattern in Yianni's code so I follow it.
Otherwise you are of course right.

Reply:
Yeah, nothing serious or flow-breaking, just code style that can be fixed later on :)

Reply:
As I already said somewhere above: This is just streamlining our container tests we we have separate module for docker and podman for every scenario with no exceptions and this will ease the future refactoring :)